src/app/shared/pipes/highlight.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'highlight',
+  standalone: true,
+})
+export class HighlightPipe implements PipeTransform {
+  transform(value: string): string {
+    const regex = /\*\*(.*?)\*\*|"(.*?)"|###/g;
+    value = value.replace(regex, (_match, p1, p2, p3) => {
+      if (p1) {
```

Comment:
could be simplified
const content = p1 | p2 
if (content) {
....
} else {
return `<br><br>`;
}